lib/SiftResponse.php

```diff
@@ -25,14 +25,9 @@ public function __construct($result, $httpStatusCode, $request, $curlErrno = 0,
         $this->rawResponse = $result;
 
         // Only attempt to get our response body if the http status code expects a body
-        if ($this->httpStatusCode >= 200 && !in_array($this->httpStatusCode, array(204, 304))) {
-            if (function_exists('json_decode')) {
-                $this->body = json_decode($result, true);
-            } else {
-                require_once(dirname(__FILE__) . '/Services_JSON-1.0.3/JSON.php');
-                $json = new Services_JSON(SERVICES_JSON_LOOSE_TYPE);
-                $this->body = $json->decode($result);
-            }
+        if ($this->httpStatusCode >= 200 && !in_array($this->httpStatusCode, [204, 304])) {
+            
+            $this->body = json_decode($result, true);
```

Comment:
Is it possible for the json_decode method to not exist?

Reply:
it is built-in in the PHP version we use.